CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.0.1] - 2020-03-12
```

Comment:
Add the actual link [0.0.1] to the end of this file.
Add the [Unreleased] link to the file

E.g.

```markdown
[0.0.1]: https://github.com/StadGent/npm_package_modal/releases/tag/0.0.1
[Unreleased]: https://github.com/StadGent/npm_package_modal/compare/master...develop
```

Reply:
Haven't actually released anything yet. I'll remove it.